### PR TITLE
fix(air-discount-scheme): Change hvk to x for non-binary individuals

### PR DESCRIPTION
**Pull request by kksteini**
# Why
'x' is preferred over 'hvk'

## Checklist:

- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] Formatting passes locally with my changes
- [X] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/air-discount-scheme/web/screens/Admin/components/Panel/Panel.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ function Panel({ flightLegs }: PropTypes) {
       return 'karlmaður'
     } else if (gender === 'kvk') {
       return 'kvenmaður'
-    } else if (gender === 'hvk') {
+    } else if (gender === 'x') {
       return 'kynsegin/annað'
     }
     return 'manneskja'
```
Comment (kristofer-lp8f) [inline apps/air-discount-scheme/web/screens/Admin/components/Panel/Panel.tsx:31]: Switch statement?

**Review by kristofer-lp8f — CHANGES_REQUESTED** [review]
Maybe Switch and shared enums?

**Review by eirikurn — APPROVED** [review]

File: apps/air-discount-scheme/backend/src/app/modules/flight/dto/body.dto.ts [hunk 2]
```diff
@@ -96,8 +96,8 @@ export class GetFlightLegsBody implements FlightLegsInput {
   age?: RangeInput
 
   @IsOptional()
-  @IsEnum(['kk', 'kvk', 'hvk'])
-  gender?: 'kk' | 'kvk' | 'hvk'
+  @IsEnum(['kk', 'kvk', 'x'])
```
[the remark this answers is not in the document]
Comment (kksteini) [reply apps/air-discount-scheme/backend/src/app/modules/flight/dto/body.dto.ts]: Yeah why not

File: apps/air-discount-scheme/backend/src/app/modules/nationalRegistry/nationalRegistry.service.ts [hunk 3]
```diff
@@ -221,13 +221,13 @@ export class NationalRegistryService {
...
-  private mapGender(genderId: string): 'kk' | 'kvk' | 'hvk' | 'óvíst' {
+  private mapGender(genderId: string): 'kk' | 'kvk' | 'x' | 'óvíst' {
     if (['1', '3'].includes(genderId)) {
       return 'kk'
     } else if (['2', '4'].includes(genderId)) {
       return 'kvk'
     } else if (['7', '8'].includes(genderId)) {
-      return 'hvk'
+      return 'x'
     }
```
[the remark this answers is not in the document]
Comment (kksteini) [reply apps/air-discount-scheme/backend/src/app/modules/nationalRegistry/nationalRegistry.service.ts]: Why not? :smile:

**Review by kristofer-lp8f — APPROVED** [review]

**Pull request merged**